TrackerConditions/test/TrkPreampStrawRandomize.py

```diff
@@ -3,34 +3,54 @@
 import sys
 import numpy as np
 
+NPlanes = 36
```

Comment:
These numbers can be obtained as StrawId::_nplanes;, etc, please use those instead.

Reply:
Otherwise the PR looks good.

Reply:
Since this is python using StrawId.hh is problematic, outside the scope of this PR.  I'll accept it for now.